Uploader_Test/packages.config

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Castle.Core" version="4.3.0" targetFramework="net461" />
```

Comment:
unused?

Reply:
moq depends on that according to nuget

Reply:
Yea, but why do you add it? nuget should resolve it recursively :dancer:

Reply:
I didn't. VS did and I can't uninstall it from the VS UI because it says Moq needs this :P

Reply:
Funny... I mostly edit the file by hand :P